shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt

```diff
@@ -392,12 +402,12 @@ object JsonLdUtils {
         return if (mediaType == MediaType.APPLICATION_JSON)
             JsonLdProcessor.compact(jsonLdEntity.properties, mapOf(JSONLD_CONTEXT to allContexts), JsonLdOptions())
                 .minus(JSONLD_CONTEXT)
-                .mapValues(restoreGeoPropertyValue())
+        // .mapValues(restoreGeoPropertyValue())
```

Comment:
??

Reply:
same I forget to remove the comment line